aospy/test/test_data_loader.py

```diff
@@ -66,6 +66,20 @@ def test_rename_grid_attrs_ds(self):
         ds = rename_grid_attrs(self.ds)
         assert LAT_STR in ds
 
+    def test_rename_grid_attrs_dim_no_coord(self):
```

Comment:
Whatever you decide to do in ```rename_grid_attrs``` just make sure to update the test to reflect the logic change (e.g. make sure attributes are copied over and / or scalar grid attribute DataArrays that are not associated with dimensions are not modified).